test/rust/main.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Why are this file and test/rust/src/main.rs necessary?

Reply:
I'm not sure that they are necessary. They currently exist in the master branch so the existence of the files is a carryover of previous work but it looks like for some reason I added a couple blank lines. I don't recall actually modifying these files manually so I'm guessing the line was added by my IDE?

I just tried deleting these two files and the tests still pass.

I'll remove the blank lines and commit. Think we should remove these files in a new PR?

Reply:
We can leave them, I thought you were adding them now (the GitHub interface confused me).

Reply:
Cool - I was confused at first too. I'm still somewhat confused how those changes ended up there in the first place 🙃